index.js

```diff
@@ -170,6 +175,14 @@ PersonaClient.prototype.validateHTTPBearerToken = function (req, res, next, scop
                 "error_description": "The token is invalid or has expired"
             });
             break;
+        case ERROR_TYPES.INSUFFICIENT_SCOPE:
+            res.status(403);
+            res.set("Connection", "close");
+            res.json({
+                "error": "insufficient_scope",
+                "error_description": "The token has insufficient scope"
```

Comment:
I think a better message would be "The supplied token is missing a required scope".